src/components/IssueController.jsx

```diff
@@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
           //console.log("Category: " + JSON.stringify(category));
           return(
             <div key={category} className="issue__category">
-              <h2 className="section-header">{category}</h2>
+              <h2 className="section-header">{category.replace('-', ' ')}</h2>
```

Comment:
Actually it wouldn't be. It might seem like overkill but perhaps we could make categories have a `slug` and `name` field. What do you think @Goldsmith94 ? It seems slightly more correct than having to fix names inline like this.